hoot-core/src/main/cpp/hoot/core/algorithms/WayJoiner.cpp

```diff
@@ -310,6 +310,9 @@ void WayJoiner::joinWays(const WayPtr &parent, const WayPtr &child)
     parent->setNodes(child_nodes);
     parent->addNodes(parent_nodes);
   }
+  //  Keep the conflated status in the parent if the child being merged is conflated
+  if (parent->getStatus() == Status::Conflated || child->getStatus() == Status::Conflated)
+    parent->setStatus(Status::Conflated);
```

Comment:
This could be simplified to remove the parent check but I think it is more easily understood with both in there.  If either way is set to `Status::Conflated` then the joined way should be too.